HelloWorld_ST_Sensors/user/internet/internet.cpp

```diff
@@ -9,6 +9,14 @@
 #include "access_point.h"
 #include "wifi.h"
 
+#define DEBUG_PRINTF 1
```

Comment:
Make 0

Reply:
This happens only 1 time.
The prints associated with it are small.
And since this WiFi init module blocks the code flow, we need these print indications.